themes/default/views/find/Results/docx_results.php

```diff
@@ -159,10 +158,10 @@
 
 			} elseif ($vs_display_text = $t_display->getDisplayValue($vo_result, $vn_placement_id, array_merge(array('request' => $this->request, 'purify' => true), is_array($va_info['settings']) ? $va_info['settings'] : array()))) {
 
-                $textrun = $contentCell->createTextRun();
-				$textrun->addText($va_info['display'].': ', $styleBundleNameFont);
-		        $textrun->addText(
-					html_entity_decode(strip_tags(br2nl($vs_display_text)), ENT_QUOTES | ENT_HTML5),
+				$textrun = $contentCell->addTextRun();
```

Comment:
`createTextRun()` is deprecated and simply returns the output of `addTextRun()`.